src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -26,11 +26,22 @@
     @SuppressFBWarnings("URF_UNREAD_FIELD")
     private int cpuCount;
 
+    @SuppressFBWarnings("URF_UNREAD_FIELD")
+    private String scheduler;
+
     public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount) {
+        this(vmName, image, baseImage, configTemplate, cpuCount, null);
+    }
+
+    public ConfigurationRequest(String vmName, String image, String baseImage, String configTemplate, int cpuCount,
+            String scheduler) {
         this.vmName = vmName;
         this.image = image;
         this.baseImage = baseImage;
         this.configTemplate = configTemplate;
         this.cpuCount = cpuCount;
+        if (scheduler != "config-default") {
+            this.scheduler = scheduler;
```

Comment:
`nit`: "config-default" is a good candidate for a constant that can be reused in the project. Maybe define it inside the `src/main/java/io/jenkins/plugins/orka/client` folder.
Also, you could use ternary here: `this.scheduler = scheduler != "config-default" ? scheduler: null;

Reply:
One other `nit`.
Instead of `null` or "config-default", we can set the scheduler to "" (empty string).
And then the check can be:
```
this.scheduler = StringUtils.isNotBlank(scheduler) ? scheduler : null;
```

This would eliminate the need for the special `config-default` string and you will not have to export it as a constant.
What do you think?